packages/json/src/formatter/index.ts

```diff
@@ -44,7 +44,6 @@ export function register(meta: MetaRegistry): void {
 		core.formatter.boolean,
 	)
 	meta.registerFormatter<JsonNullNode>('json:null', () => 'null')
-	meta.registerFormatter<JsonNumberNode>('json:number', core.formatter.float)
```

Comment:
I am pretty sure this should not be needed anymore, as the json number node now has a child of either long or float, which should be formatted by the core formatters already.

Reply:
"pretty sure"? Did you test this?

Reply:
no, the formatter only returns a toString for float, so I don't know if there is any observable change when it's therers when it's not there.

```ts
export const float: Formatter<FloatBaseNode> = (node) => {
	return node.value.toString()
}
```

Reply:
Ah, actually there is an issue here, it complains about not finding the json:number formatter.